openstack_controller/tests/test_unit_glance.py

```diff
@@ -334,3 +335,31 @@ def test_images_pagination(
             tags=metric['tags'],
             hostname=metric.get('hostname'),
         )
+
```

Comment:
nit- can you add a test for when these metrics are disabled?

Reply:
Testing the disabling of any `openstack.glance.image.*` (including the new `*.task.count`) metric is already done by `test_disable_glance_images_metrics` (on line 68).

Reply:
Makes sense, i missed that!

Reply:
But, are you testing when `images` config key is true and `tasks` config key is false?

Reply:
I have added a test specifically disabling `tasks`.